sites/ontvtonight.com/ontvtonight.com_us.channels.xml

```diff
@@ -34,6 +34,7 @@
 	<channel lang="en" xmltv_id="TBSEast.us" site_id="#69046986/tbs-superstation-east">TBS Superstation (East)</channel>
 	<channel lang="en" xmltv_id="truTVWest.us" site_id="#69045208/trutv-hdtv-west">truTV HDTV (West)</channel>
 	<channel lang="en" xmltv_id="WCBSDT1.us" site_id="#69021345/wcbs">CBS (WCBS)</channel>
+	<channel lang="en" xmltv_id="WCAX-TV6.us" site_id="10040857671/wcax-news-weather">3 News Now</channel>

```

Comment:
```suggestion
	<channel lang="en" xmltv_id="WCAXTV6.us" site_id="#10040857671/wcax-news-weather">3 News Now</channel>
```

Reply:
There is no such channel in the database: https://iptv-org.github.io/?q=WCAX

Reply:
Indeed. https://www.tvpassport.com/tv-listings/stations/3-news-now-wcaxtv6-burlington-vt/36205 seems to list it too.

https://wikipedia.org/wiki/WCAX-TV according to this Wikipedia page it exists since Sept. 2021